src/main/groovy/com/pstehlik/groovy/gelf4j/net/GelfChunkBuilder.groovy

```diff
@@ -0,0 +1,67 @@
+package com.pstehlik.groovy.gelf4j.net
+
+import java.nio.ByteBuffer
+
+class GelfChunkBuilder {
+
+  static final int MAX_CHUNK_COUNT = 128
+
+  static final int HEADER_LENGTH = 12
+
+  static final int CHUNK_NUMBER_POSITION = 10
+
+  ByteBuffer header
+
+  int maxChunkSize
+
+  int totalChunkCount
+
+  GelfChunkBuilder(int totalChunkCount, int maxChunkSize) {
+
+    if (totalChunkCount > MAX_CHUNK_COUNT) {
+      throw new IllegalArgumentException("Chunk count cannot exceed ${MAX_CHUNK_COUNT}: [${totalChunkCount}]")
+    }
+
+    this.maxChunkSize = maxChunkSize
+    this.totalChunkCount = totalChunkCount
+    header = ByteBuffer.allocate(HEADER_LENGTH)
+
+      .put(0x1e.byteValue())                            // The GELF magic numbers
+      .put(0x0f.byteValue())
+      .putLong(UUID.randomUUID().mostSignificantBits)   // a new random Message GUID
+      .put(0.byteValue())                               // the current chunk number
+      .put(totalChunkCount.byteValue())                 // the total chunk count
```

Comment:
The chunk headers following the new GLEF format, including smaller fields for the ID chunk number and chunk count